ilagan_andreijireh/script.js

```diff
@@ -91,4 +91,37 @@ function loadComments() {
     comments = JSON.parse(storedComments);
     displayComments();
   }
-}
\ No newline at end of file
+}
+
+document.getElementById('search-button').addEventListener('click', function() {
+  var country = document.getElementById('country-input').value;
```

Comment:
### Change the var keyword to let

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g25d183194fb_0_0